cli/cmd/vuln_container.go

```diff
@@ -442,20 +448,26 @@ func buildVulnerabilityReportTable(assessment *api.VulnContainerAssessment) stri
 		),
 	)
 
-	if vulCmdState.Details || vulCmdState.Fixable || vulCmdState.Packages {
+	if vulCmdState.Details || vulCmdState.Fixable || vulCmdState.Packages || vulFiltersEnabled() {
```

Comment:
I think we have so many filters now that we could do what you did for compliance filters, abstract them into a function.

Reply:
Here, i've distinguished filters from the other flags details and packages in vulFiltersEnabled(). I could see fixable being pulled into the vulFiltersEnabled but I don't think details and packages quite fit.

Reply:
Done for this fixible. I don't think packages and details should be pulled in.